src/lambdas/get-app-theme-color.lambda.ts

```diff
@@ -57,17 +85,31 @@ export const getButtonColorLambda = (
   if (!theme || theme === AppTheme.Basic) return undefined
 
   // by default halloween as only two options for now:
+  // if (disabled) {
+  //   return {
+  //     color: Halloween.three, // it will be the font color
+  //     borderColor: Halloween.one, // it will be the border color
+  //     backgroundColor: Halloween.one,
+  //   }
+  // }
+
   if (disabled) {
     return {
-      color: Halloween.three, // it will be the font color
-      borderColor: Halloween.one, // it will be the border color
-      backgroundColor: Halloween.one,
+      color: Christmas.five, // it will be the font color
+      borderColor: Christmas.five, // it will be the border color
+      backgroundColor: Christmas.seven,
     }
   }
 
+  // return {
+  //   color: disabled ? Halloween.three : Halloween.five,
+  //   borderColor: disabled ? Halloween.one : Halloween.five,
+  //   backgroundColor: disabled ? undefined : Halloween.two,
+  // }
+
```

Comment:
if (theme === AppTheme.Christmas) { ... }

Reply:
Handled in [8ef685e](https://github.com/ajktown/ConsistencyGPT/pull/74/commits/8ef685e54a10fbb3a6fff4f9964b248e91c083eb)

Reply:
Handled in [8ef685e](https://github.com/ajktown/ConsistencyGPT/pull/74/commits/8ef685e54a10fbb3a6fff4f9964b248e91c083eb)